granary/flickr.py

```diff
@@ -511,6 +523,18 @@ def photo_to_activity(self, photo):
         'displayName': tag.strip(),
       } for tag in photo.get('tags').split(' ') if tag.strip()]
 
+    # location is represented differently in a list of photos vs a
+    # single photo info
+    lat = photo.get('latitude') or photo.get('location', {}).get('latitude')
+    lng = photo.get('longitude') or photo.get('location', {}).get('longitude')
+    if lat and lng and float(lat) != 0 and float(lng) != 0:
+      activity['object']['location'] = {
+        'objectType': 'place',
+        'latitude': float(lat),
+        'longitude': float(lng),
+      }
+
+    self.postprocess_object(activity['object'])
```

Comment:
it seems like we should just do this in `postprocess_activity`, but i haven't elsewhere, and i suspect there was a reason, but i don't remember why and i don't see a comment. sigh. :P